openmc/settings.py

```diff
@@ -936,6 +936,17 @@ def max_tracks(self, value: int):
         cv.check_greater_than('maximum particle tracks', value, 0, True)
         self._max_tracks = value
 
+    @property
+    def get_sum_of_source_strengths(self):
+        """Gets the total source strength of all sources. Useful when
+        normalizing the tally result from model with sources that comprise of
+        multiple openmc.Source objects"""
+
+        total_strength = 0
+        for source in self.source:
+            total_strength += source.strength
+        return total_strength
```

Comment:
To me this is a simple one-liner:
```Python
total_strength = sum(src.strength for src in settings.source)
```
I don't think we need a method for every tiny operation and, to me, this is something a user can easily do. So, I'm not in favor of merging this PR. Sorry @shimwell!

Regarding the larger question of a `Sources` object, I would suggest creating an issue where we can discuss further what the intended use would be, including what behaviors are envisioned for such a class.